### PR TITLE
Minor doc changes and copyright addition

### PR DESCRIPTION
**Pull request by rounak-adhikary**

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,24 @@ limitations under the License.
...
+### Resources
+* Lifecycle Controller Attributes Resource
+* System Attributes Resource
+* iDRAC Firmware Update Resource
+
+### Others
+N/A
+
+## Enhancements
+* Added storage controller details to storage datasource
```
Comment (Sakshi-dell) [inline CHANGELOG.md:35]: storage volume resource is also enhanced

File: README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ The Terraform Provider for RedFish is released and licensed under the MPL-2.0 li
 
 | **Terraform Provider** | **iDRAC Version** | **OS** | **Terraform** | **Golang** |
 |---------------------|-----------------------|-------|--------------------|--------------------------|
-| v1.1.0 | 5.x <br> 6.x <br> 7.x | ubuntu22.04 <br> rhel8.x <br> rhel9.x | 1.4.x <br> 1.5.x <br> | 1.20
+| v1.1.0 | 5.x <br> 6.x <br> 7.x | ubuntu22.04 <br> rhel9.x | 1.5.x <br> 1.6.x | 1.21
```
Comment (Sakshi-dell) [inline README.md:52]: version needs to be updated to 1.2.0

File: examples/resources/redfish_storage_volume/import.sh [hunk 3]
```diff
@@ -15,4 +15,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+# Odata ID of all available volumes on a storage controller can be fetched by running the following GET request on the iDRAC
```
Comment (Sakshi-dell) [inline examples/resources/redfish_storage_volume/import.sh:18]: Shouldn't this be rather added as notes in templates?

File: README.md (hunk 2 above)
Comment (gokul-srivathsan) [reply README.md:52]: The 2nd column header should be iDrac9 Firmware Version.

File: CHANGELOG.md [hunk 4]
```diff
@@ -18,6 +18,24 @@ limitations under the License.
...
+The release supports resources and data sources mentioned in the Features section for RedFish.
+## Features
+
+### Resources
+* Lifecycle Controller Attributes Resource
+* System Attributes Resource
+* iDRAC Firmware Update Resource
+
+### Others
+N/A
```
Comment (gokul-srivathsan) [inline CHANGELOG.md:32]: Add one point about additional 'Import' functionality.

**Review by gokul-srivathsan — APPROVED** [review]
LGTM

**Review by Krishnan-Priyanshu — APPROVED** [review]

**Review by Sakshi-dell — APPROVED** [review]

**Pull request merged**